src/main/webapp/script.js

```diff
@@ -1,4 +1,43 @@
+// TODO: Retrieve data from backend
```

Comment:
Just making sure but these are the hard coded stats? I would suggest mentioning that these are hardcoded and will be changed later in the next sprint.